### PR TITLE
Dev/atomsbase integration

### PR DESCRIPTION
**Pull request by vcanogil**

### CODE REVIEW

**Comment by vcanogil** [comment]
This is ready to merged whenever you are ready.

**Review by berquist — DISMISSED** [review]
Minor stuff, I can confirm this is working.

File: src/ab_integration.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+into a list of AtomsBase atom objects.
+
+# Arguments
+- `mol::String`: File supported by `ccread` function
+
+# Returns
+A list of AtomsBase atom objects
+"""
+function get_atom_objects(mol::String)
+    mol = ccread(mol)
```
[the remark this answers is not in the document]
Comment (vcanogil) [reply src/ab_integration.jl:40]: Didn't notice. Will fix that.

File: test/runtests.jl [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 @testset "Cclib.jl" begin
+    test_file = "./data/uracil_two.xyz"
 
     # Check that input is read correctly
-    @test ccread("./data/uracil_two.xyz")["natom"] == 12
-    @test ccread("./data/uracil_two.xyz")["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
-    @test ccread("./data/uracil_two.xyz")["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
+    @test ccread(test_file)["natom"] == 12
+    @test ccread(test_file)["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
+    @test ccread(test_file)["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
```
Comment (shivupa) [inline test/runtests.jl:14]: I think eric was saying can we do
```
    test_data = ccread(test_file)
    @test test_data["natom"] == 12
    @test test_data["atomnos"] == [7, 6, 1, 8, 7, 6, 8, 6, 6, 1, 1, 1]
    @test test_data["atomcoords"] == [5.435 4.091 6.109 3.647 3.253 3.636 2.79 5.065 5.906 2.243 5.448 7.003;
```

This prevents reading the file multiple times

**Review by shivupa — CHANGES_REQUESTED** [review]
more minor changes sorry

File: test/runtests.jl (hunk 2 above)
Comment (vcanogil) [reply test/runtests.jl:14]: done

**Review by shivupa — APPROVED** [review]

**Pull request merged**